dev/devicelab/lib/framework/ab.dart

```diff
@@ -30,6 +32,48 @@ class ABTest {
     _addResult(result, _bResults);
   }
 
+  static void accumulateLengths(List<int> lengths, List<String> results) {
+    for (int i = 0; i < lengths.length; i++) {
+      if (results[i] != null) {
+        final int len = results[i].length;
+        if (lengths[i] < len) {
+          lengths[i] = len;
+        }
+      }
+    }
+  }
+
+  static void formatResult(StringBuffer buffer,
+                           List<int> lengths,
+                           List<FieldJustification> aligns,
+                           List<String> values) {
+    for (int i = 0; i < lengths.length; i++) {
```

Comment:
same nit: rename `i` to `column` to clarify that it's the index for table column.